app/src/main/java/social/entourage/android/new_v8/events/details/feed/FeedFragment.kt

```diff
@@ -267,14 +267,35 @@ class FeedFragment : Fragment() {
     private fun initializePosts() {
         binding.postsNewRecyclerview.apply {
             layoutManager = LinearLayoutManager(requireContext())
-            adapter = GroupPostsAdapter(newPostsList, eventId, event.member, event.title)
+            adapter = PostAdapter(
+                newPostsList,
+                ::openCommentPage
+            )
         }
         binding.postsOldRecyclerview.apply {
             layoutManager = LinearLayoutManager(requireContext())
-            adapter = GroupPostsAdapter(oldPostsList, eventId, event.member, event.title)
+            adapter = PostAdapter(
+                oldPostsList,
+                ::openCommentPage
+            )
         }
     }
 
+    private fun openCommentPage(post: Post, shouldOpenKeyboard: Boolean) {
+        context?.startActivity(
+            Intent(context, EventCommentActivity::class.java)
+                .putExtra(
+                    Const.ID,
+                    eventId
+                )
+                .putExtra(Const.POST_ID, post.id)
+                .putExtra(Const.POST_AUTHOR_ID, post.user?.userId)
+                .putExtra(Const.SHOULD_OPEN_KEYBOARD, shouldOpenKeyboard)
+                .putExtra(Const.IS_MEMBER, event.member)
+                .putExtra(Const.NAME, event.title)
```

Comment:
Intent(context, Activity::class.java)
            .putExtras(bundleOf(
                "coucou" to "truc",
                "salut" to "bidule",
                "bonjour" to "chouette"
            ))